lib/transactional.pm

```diff
@@ -156,14 +169,19 @@ sub trup_call {
     my ($cmd, %args) = @_;
     $args{timeout} //= 180;
     $args{exit_code} //= 0;
+    my $tunneled = get_var("TUNNELED") && testapi::is_serial_terminal();    # Don't pipe stuff to serialterminal in TUNNELED mode
 
     # Always wait for rollback.service to be finished before triggering manually transactional-update
     ensure_rollback_service_not_running();
 
-    my $script = "transactional-update $cmd > /dev/$serialdev";
+    my $script = "transactional-update $cmd";
+    $script .= " > /dev/$serialdev" unless ($tunneled);
```

Comment:
This should work in tunnelled mode as in tunnelled mode the `/dev/$serialdev` should point to `/dev/sshserial`

Reply:
This change should be not part of this PR. I'll remove it, this is not intended to be here.